basyx.aasenvironment/basyx.aasenvironment.component/src/main/resources/rbac_rules.json

```diff
@@ -0,0 +1,235 @@
+[
```

Comment:
there are some doubled rolenames, is this intentional? maybe if they don't need to be the same name, give them different ones, that would make it more easy to tell them apart

Reply:
Thanks for your detailed review.
The reason for this doubled rolenames or basically multiple rules with the same role name is because one role could be reused in multiple target information. Hence, for example the role **basyx-reader** is for AAS as well as Submodel target information.

```
[
  {
    "role": "basyx-reader",
    "action": "READ",
    "targetInformation": {
      "@type": "aas",
      "aasId": "*"
    }
  },
  {
    "role": "basyx-reader",
    "action": "READ",
    "targetInformation": {
      "@type": "submodel",
      "submodelId": "*",
      "submodelElementIdShortPath": "*"
    }
  }
]
```

Please correct me if I misunderstood here while answering.